katello-selinux-enable

```diff
@@ -3,6 +3,7 @@ set +e
 
 TMP=$(mktemp -t katello-selinux-enable.XXXXXXXXXX)
 trap "rm -rf '$TMP'" EXIT INT TERM
+LOG=/var/log/foreman-selinux-install.log
```

Comment:
should this be `katello-selinux-enable.log`?

Reply:
I see foreman-selinux uses the same log (https://github.com/theforeman/foreman-selinux/blob/develop/foreman-selinux-enable#L7) and my assumption is this would append to that log and keep our ecosystem of selinux changes in a single place. That sounds OK to me.

@lzap ?

Reply:
Yeah I thought the same, it appends and adds also `$0` so it's clear what is logging there.

Reply:
wfm!